core/lib/Thelia/Config/Resources/form.xml

```diff
@@ -114,9 +114,10 @@
         <form name="thelia.admin.area.create" class="Thelia\Form\Area\AreaCreateForm"/>
         <form name="thelia.admin.area.modification" class="Thelia\Form\Area\AreaModificationForm"/>
         <form name="thelia.admin.area.country" class="Thelia\Form\Area\AreaCountryForm"/>
+        <form name="thelia.admin.area.delete.country" class="Thelia\Form\Area\AreaDeleteCountryForm"/>
         <form name="thelia.admin.area.postage" class="Thelia\Form\Area\AreaPostageForm"/>
 
-        <form name="thelia.shopping_zone_area" class="Thelia\Form\ShippingZone\ShippingZoneAddArea"/>
+        <form name="thelia.shipping_zone_area" class="Thelia\Form\ShippingZone\ShippingZoneAddArea"/>
```

Comment:
This is BC for modules. I think we should keep both and remove the typo in another version